app.js

```diff
@@ -10,40 +11,103 @@ app.use(express.json());
 
 
 // Iteration 1 - Connect to MongoDB
-// DATABASE CONNECTION
 
+const mongoose = require('mongoose')
+
+const MONGODB_URI = "mongodb://127.0.0.1:27017/express-mongoose-recipes-dev";
+
+mongoose
+  .connect(MONGODB_URI)
+  .then((x) => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
+  .catch((err) => console.error("Error connecting to mongo", err));
 
 
 // ROUTES
-//  GET  / route
+//  GET  / route - This is just an example route
 app.get('/', (req, res) => {
-    res.send("<h1>Lab | Express Mongoose Recipes</h1>");
+    res.send("<h1>LAB | Express Mongoose Recipes</h1>");
 });
 
-
-//  Iteration 4 - Create recipe route
+//  Iteration 3 - Create a Recipe route
 //  POST  /recipes route
 
-
-//  Iteration 5 - Read all recipes
+app.post('/recipes',(req,res)=>{
+    Recipe.create({
+        title:req.body.title,
+        instructions:req.body.instructions,
+        level:req.body.level,
+        ingredients:req.body.ingredients,
+        image:req.body.image,
+        duration:req.body.duration,
+        isArchived:req.body.isArchived,
+        created:req.body.created
+    })
```

Comment:
Something that can help other person that dont complete know which is the Schema of the Recipies could be so mething like this, is just for readibilty
```suggestion
const { title, instructions, level, ingredients, image ,duration, isArchived, created} = req.body
  Recipe.create({
    title,
    instructions,
    level,
    ingredients,
    image,
    duration,
    isArchived,
    created,
  })
```